libs/infra-nest-server/src/lib/bootstrap.ts

```diff
@@ -84,6 +84,8 @@ export const createApp = async ({
     },
   )
 
+  app.enableVersioning()
```

Comment:
Good catch. NestJS documentation actually has a warning box with 
```
If versioning is enabled for the application but the controller or route does not specify the version, any requests to that controller/route will be returned a 404 response status. Similarly, if a request is received containing a version that does not have a corresponding controller or route, it will also be returned a 404 response status.
```
So I will add this as an opt-in parameter to the bootstrap function for now.